src/newcharacter.cpp

```diff
@@ -2293,11 +2296,19 @@ tab_direction set_description(WINDOW *w, player *u, const bool allow_reroll, poi
         const std::string action = ctxt.handle_input();
 
         if (action == "NEXT_TAB") {
-            if( OPTIONS["POINT_DISTRIBUTION"] != "freeform" && !points.is_valid() ) {
-                popup(_("Too many points allocated, change some features and try again."));
+            if (OPTIONS["POINT_DISTRIBUTION"] != "freeform" && !points.is_valid() ) {
+                if (points.skill_points_left() < 0 ) {
+                        popup(_("Too many points allocated, change some features and try again."));
```

Comment:
I'll add here for future reference that I can't get rid of the skill points error check due to the order the errors are checked. Having a negative total points will trigger the first error message always, so it has to be a general one. The last "else" is only there for sanity reasons (I think that's the correct term...).